crates/ide-assists/src/handlers/generate_delegate_methods.rs

```diff
@@ -106,7 +107,10 @@ pub(crate) fn generate_delegate_methods(acc: &mut Assists, ctx: &AssistContext<'
             |edit| {
                 // Create the function
                 let method_source = match method.source(ctx.db()) {
```

Comment:
```suggestion
                let method_source = match ctx.sema.source(method) {
```
We can do this instead of the extra `parse_or_expand`, this will do the caching for us then